src/Microsoft.ML.Transforms/Text/StopWordsRemovingTransformer.cs

```diff
@@ -1095,9 +1095,9 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
     /// | -- | -- |
     /// | Does this estimator need to look at the data to train its parameters? | No |
     /// | Input column data type | Vector of [Text](xref:Microsoft.ML.Data.TextDataViewType) |
-    /// | Output column data type | Unknown-sized vector of [Text](xref:Microsoft.ML.Data.TextDataViewType) |
+    /// | Output column data type | variable-sized vector of [Text](xref:Microsoft.ML.Data.TextDataViewType) |
     ///
-    /// The resulting <xref:Microsoft.ML.Transforms.Text.CustomStopWordsRemovingTransformer/> creates a new column, named as specified by the output column name parameter, and
```

Comment:
>/> [](start = 91, length = 2)

why not just change /> to > and make it autolink

Reply:
+1

---
In reply to: [277886864](https://github.com/dotnet/machinelearning/pull/3552#discussion_r277886864) [](ancestors = 277886864)